### ISSUE TITLE
"CMakeLists.txt" not in the source directory

### ISSUE THREAD

**Issue by umbe1987** [potree/PotreeConverter#176]
I am trying to build lastools on Windows 8.1 following the instructions in https://github.com/potree/PotreeConverter. At line "cmake -G "Visual Studio 14 2015 Win64" ../" I receive the following error:
CMake Error: The source directory "MYPATH/LAStools/master" does not appear to contain CMakeLists.txt.
Looking at the "master" dir, the "CMakeLists.txt" is not present. I used the code "git clone https://github.com/m-schuetz/LAStools.git master" to copy the fork of lastools with cmake, but that seems not to contain the CMakeLists.txt file needed.

**Comment by m-schuetz** [comment]
Hi,

The CMakeLists.txt is in the laszip directory. 
Have you called the cmake command from within the "lastools/laszip/build" directory?

**Comment by umbe1987** [comment]
Some little steps on... I added an empty "CMakeLists.txt" file in the master folder. Now it seems it's trying to compile, but the error is now that the compiler is unknown and the configuration is incomplete. You can see the error log in the attached file.

[CMakeError.txt](https://github.com/potree/PotreeConverter/files/64049/CMakeError.txt)

**Comment by umbe1987** [comment]
Thanks for the reply.
I tried it and it is now trying to compile, but it gives me this error:
  -- The C compiler identification is unknown
  -- The CXX compiler identification is unknown
  CMake Error at CMakeLists.txt:3 (project):
    No CMAKE_C_COMPILER could be found

  CMake Error at CMakeLists.txt:3 (project):
    No CMAKE_CXX_COMPILER could be found

  -- Configuring incomplete, errors occurred!

I also attached the error log.
The build path should be updated in the installation instructions (it is correct for Linux, but not for Windows).

Thanks for the effort and reply!
[CMakeError.txt](https://github.com/potree/PotreeConverter/files/64057/CMakeError.txt)

**Comment by m-schuetz** [comment]
Not sure about that error. Can you cmake any other project? Sounds to me like cmake can't find the correct compiler to use. Is Visual Studio 2015 installed?
Maybe googling that error can help you there.

**Comment by umbe1987** [comment]
Hi,

I installed "Visual C++ Redistributable for Visual Studio 2015" from the download site you provided (both the 32bit and 64bit versions). Unfortunately I have not experience in compiling programs, so I am not able to test if this error ocurs on other project...

Anyway, I tried to pass the compiler path to cmake on Ubuntu, as suggested in http://stackoverflow.com/questions/9699930/cmake-complains-the-cxx-compiler-identification-is-unknown.

The output is now different (see below), but still the configuration is incomplete.

minorau@minorau-VirtualBox:~/lastools/master/LASzip/build$ cmake -DCMAKE_CXX_COMPILER=/usr/bin/gcc -DCMAKE_BUILD_TYPE=Release ..
-- The CXX compiler identification is unknown
-- Check for working CXX compiler: /usr/bin/gcc
-- Check for working CXX compiler: /usr/bin/gcc -- broken
CMake Error at /usr/share/cmake-2.8/Modules/CMakeTestCXXCompiler.cmake:54 (message):
  The C++ compiler "/usr/bin/gcc" is not able to compile a simple test
  program.

  It fails with the following output:

   Change Dir: /home/minorau/lastools/master/LASzip/build/CMakeFiles/CMakeTmp  
  Run Build Command:/usr/bin/make "cmTryCompileExec1120029000/fast"

  /usr/bin/make -f CMakeFiles/cmTryCompileExec1120029000.dir/build.make
  CMakeFiles/cmTryCompileExec1120029000.dir/build

  make[1]: Entering directory
  `/home/minorau/lastools/master/LASzip/build/CMakeFiles/CMakeTmp'

  /usr/bin/cmake -E cmake_progress_report
  /home/minorau/lastools/master/LASzip/build/CMakeFiles/CMakeTmp/CMakeFiles 1

  Building CXX object
  CMakeFiles/cmTryCompileExec1120029000.dir/testCXXCompiler.cxx.o

  /usr/bin/gcc -o
  CMakeFiles/cmTryCompileExec1120029000.dir/testCXXCompiler.cxx.o -c
  /home/minorau/lastools/master/LASzip/build/CMakeFiles/CMakeTmp/testCXXCompiler.cxx


  gcc: error trying to exec 'cc1plus': execvp: No such file or directory

  make[1]: Leaving directory
  `/home/minorau/lastools/master/LASzip/build/CMakeFiles/CMakeTmp'

  make[1]: ***
  [CMakeFiles/cmTryCompileExec1120029000.dir/testCXXCompiler.cxx.o] Error 1

  make: *** [cmTryCompileExec1120029000/fast] Error 2

**Comment by umbe1987** [comment]
Ok, I went on with the Ubuntu installation.
Apparently the problem was I did not install g++.
Now it compiled correctly. Now the "make" command gives me an error, but I think I will not talk about it in this issue.
Anyway, the Windows problem might be due to a similar cause. Would installing "Visual C++ Redistributable for Visual Studio 2015" be not sufficient?
I am running commands from a Powershell window, is this what I am supposed to?

For the other linux related problem I will try to find a solution myself, hopefully I'll be successfull!
Thanks in advance.
